larq/quantizers.py

```diff
@@ -601,6 +603,86 @@ def get_config(self):
         return {**super().get_config(), "k_bit": self.precision}
 
 
+@utils.register_alias("dorefa_kernel_quantizer")
+@utils.register_keras_custom_object
+class DoReFaKernel(DoReFa):
+    r"""Instantiates a serializable k_bit kernel quantizer as in the DoReFa paper.
```

Comment:
When `k=1`, the DoReFa weight quantizer equals `SteSign`. Is this currently handled correctly?

Reply:
It is, because the weight quantizer only adds a preprocessor for the activation quantizer.